activation/nipost.go

```diff
@@ -406,6 +406,21 @@ func (nb *NIPostBuilder) submitPoetChallenges(
 		}
 	}
 
+	misconfiguredRegistrations := make(map[string]nipost.PoETRegistration)
+	for addr := range registrationsMap {
+		if reg, ok := existingRegistrationsMap[addr]; !ok {
+			misconfiguredRegistrations[addr] = reg
+		}
+	}
+
+	if len(misconfiguredRegistrations) != 0 {
+		nb.logger.Warn(
+			"Invalid PoET registrations found. Local state contains PoETs not in node config",
+			zap.Strings("registrations_addresses", maps.Keys(misconfiguredRegistrations)),
+			log.ZShortStringer("smesherID", nodeID),
+		)
```

Comment:
It sounds like the local state was broken, while the config is suspicious. How about rephrasing:
```suggestion
		nb.logger.Warn(
			"Found existing registrations for poets not listed in the config. Will not fetch proof from them.",
			zap.Strings("registrations_addresses", maps.Keys(misconfiguredRegistrations)),
			log.ZShortStringer("smesherID", nodeID),
		)
```